spike_main/spike.cc

```diff
@@ -376,6 +378,7 @@ int main(int argc, char** argv)
         exit(-1);
      }
   });
+  parser.option(0, "blocksz", 1, [&](const char* s){blocksz = strtoull(s, 0, 0);});
```

Comment:
How about checking whether the block size is a power of 2 somewhere? See `processor_t::check_pow2` for argument checking.

Reply:
I have sent email to David to discuss about the unsolved problems, including the reasonable range for cache block size. There is no reply currently. Anyway, I agree it should be power of 2. I'll update it later.